packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -127,32 +142,27 @@ export const registerPostTypeSchema =
 			.resolveSelect( coreStore )
 			.getCurrentTheme();
 
+		const isDeisgnPostType = DESIGN_POST_TYPES.includes( postType );
 		const actions = [
 			postTypeConfig.viewable ? viewPost : undefined,
 			!! postTypeConfig.supports?.revisions
 				? viewPostRevisions
 				: undefined,
 			// @ts-ignore
 			globalThis.IS_GUTENBERG_PLUGIN
-				? ! [ 'wp_template', 'wp_block', 'wp_template_part' ].includes(
-						postTypeConfig.slug
-				  ) &&
-				  canCreate &&
-				  duplicatePost
+				? ! isDeisgnPostType && canCreate && duplicatePost
```

Comment:
This isn't the same logic as before, right? It includes now the navigation post type. I'm not sure we need to introduce a new concept ("design_post_types"). It looks like checking for specifics would be simpler.